test/test-applications/integrations/Directory.Build.props

```diff
@@ -24,31 +22,4 @@
     <!-- Hide warnings for EOL .NET Core targets (e.g. netcoreapp3.0) -->
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
   </PropertyGroup>
-
-  <ItemGroup Condition="'$(ExcludeManagedProfiler)' != 'true' and
-                        '$(LoadManagedProfilerFromProfilerDirectory)' != 'true' ">
-    <ProjectReference Include="$(MSBuildThisFileDirectory)\..\..\..\src\OpenTelemetry.ClrProfiler.Managed\OpenTelemetry.ClrProfiler.Managed.csproj" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(ExcludeNativeProfiler)' != 'true'">
-    <None Include="$(ProfilerOutputDirectory)\*.dll;$(ProfilerOutputDirectory)\*.so;$(ProfilerOutputDirectory)\*.pdb"
-          CopyToOutputDirectory="Always"
-          CopyToPublishDirectory="Always"
-          Link="profiler-lib\%(RecursiveDir)\%(Filename)%(Extension)" />
-    <Content Include="$(MSBuildThisFileDirectory)\..\..\..\integrations.json"
-             CopyToOutputDirectory="Always"
-             CopyToPublishDirectory="Always"
-             Link="profiler-lib\integrations.json" />
-  </ItemGroup>
-
-  <Target Name="AfterBuildCopyManagedProfiler" AfterTargets="AfterBuild"
-          Condition=" '$(ExcludeManagedProfiler)' != 'true' AND '$(LoadManagedProfilerFromProfilerDirectory)' == 'true'">
-    <ItemGroup>
-      <!-- Subfolders of the output directory should each be a target framework -->
-      <ManagedProfilerFiles Include="$(ManagedProfilerOutputDirectory)\**\*.dll" Exclude="$(ManagedProfilerOutputDirectory)\*\runtimes\**\*.dll" />
-      <ManagedProfilerFiles Include="$(ManagedProfilerOutputDirectory)\**\*.pdb" Exclude="$(ManagedProfilerOutputDirectory)\*\runtimes\**\*.pdb" />
-    </ItemGroup>
-
-    <Copy SourceFiles="@(ManagedProfilerFiles)" DestinationFolder="$(OutputPath)profiler-lib\%(RecursiveDir)" />
-  </Target>
 </Project>
```

Comment:
I guess that wouldn't be good for debugging. Then you would need again full sln.

Reply:
I think we should be aware that the Nuke build are the primary way to build/run/test stuff. 

BTW take a look at https://github.com/DataDog/dd-trace-dotnet/blob/6cfaa55336f52ef09afe87bc093dd207738d9dab/tracer/build/_build/Build.Utilities.cs#L118

Reply:
This is a limitation in the Datadog repo that we just live with, for the time being. Your workflow has to be `build BuildTracerHome` and then you can Debug a test application. We haven't implemented a better solution in that repo yet, but it's been tolerable so far

Reply:
> BTW take a look at https://github.com/DataDog/dd-trace-dotnet/blob/6cfaa55336f52ef09afe87bc093dd207738d9dab/tracer/build/_build/Build.Utilities.cs#L118

The `RunDotNetSample` has been a good way to run samples as well, though debugging through VS means you'll have to attach to process instead of starting the process with a debugger already attached.